pyproject.toml

```diff
@@ -63,12 +63,113 @@ distance-dirty = "{next_version}.dev{distance}+d{build_date:%Y%m%d%H%M}"
 [tool.versioningit.write]
 file = "reduction/lr_reduction/_version.py"
 
+####################################################################
+#  CREATING CONDA ENVIRONMENT AND BUILDING CONDA PACKAGE WITH PIXI
+####################################################################
+[tool.pixi.workspace]
+preview = ["pixi-build"]
+channels = [
+  "mantid-ornl",
+  "conda-forge",
+  "https://prefix.dev/pixi-build-backends"
+]
+platforms = ["linux-64"]  # add additional platforms as needed
+
+# Conda packages (no PyPi) to be enumerated within the conda package to be built.
+[tool.pixi.package.run-dependencies]
+python = ">=3.10"
+lmfit = "*"
+mantid = ">=6.12.0.2"
+matplotlib = ">=3.9.4,<4"
+
+# Conda packages to be installed in the environments. Includes all the run-dependencies.
+[tool.pixi.dependencies]
+argcomplete = ">=3.6.2,<4"
+hatchling = ">=1.27.0,<2"
+#   run-dependiencies replicated here
+python = ">=3.10"
+lmfit = "*"
+mantid = ">=6.12.0.2"
+matplotlib = ">=3.9.4,<4"
+conda-verify = ">=3.4.2,<4"
+conda-build = ">=24.5.1,<25"
+
+# Pypi packages to be installed in the environments with `pip`
+[tool.pixi.pypi-dependencies]
+lr_reduction = { path = ".", editable = true }  # editable install of lr_reduction
+toml-cli = "*"
+
+[tool.pixi.package]
+name = "lr_reduction"
+version = "0.0.0"  # a placeholder overwritten when Pixi task "sync-version" is executed
+
+[tool.pixi.package.build]
+backend = { name = "pixi-build-python", version = "0.1.*" }
+
+[tool.pixi.package.host-dependencies]
+hatchling= ">=1.27.0,<2"
+versioningit = "*"
+
+
+###################################
+#  PIXI FEATURES AND ENVIRONMENTS
+###################################
+[tool.pixi.feature.installer.dependencies]  # necessary to install and inspect
+hatch= "*"
+pip = "*"
+versioningit = "*"
+conda-tree = "*"
+
+[tool.pixi.feature.developer.dependencies]  # necessary for development and testing
+pre-commit = "*"
+ruff = "*"
+codecov = "*"
+pytest = "*"
+pytest-cov = "*"
+
+[tool.pixi.feature.package.dependencies]  # necessary to build the conda package
+boa = "*"
```

Comment:
Please drop boa because you aren't using it

Reply:
removed